admin/pkg/authtoken/authtoken.go

```diff
@@ -131,6 +131,14 @@ func (t *Token) SecretHash() []byte {
 	return hashed[:]
 }
 
+// Prefix returns a safe, partial display string for the token, e.g., "rill_usr_abcdefghij"
+func (t *Token) Prefix() string {
+	payload := make([]byte, 40)
+	copy(payload[0:11], t.ID[:])
```

Comment:
Just copy the whole ID – even if not used, makes it easier to change the prefix length and reduces chance of edge cases in base62 we maybe didn't think about
```suggestion
	payload := make([]byte, 40)
	copy(payload[0:16], t.ID[:])
```